config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customers#create"
```

Comment:
I don't love this routing because the path and controller action suggests that we're creating a new customer, not adding an item to an existing customer. You could instead have this routing be `post "/customers/:id/items", to: "customer_items#create"` as one option.